src/Analyzer.Utilities/DiagnosticHelpers.cs

```diff
@@ -16,6 +16,22 @@ public static class DiagnosticHelpers
             DiagnosticSeverity.Warning;
 #endif
 
+        public const bool EnabledByDefaultIfNotBuildingVSIX =
+#if USE_INTERNAL_IOPERATION_APIS // Building Analyzer VSIX
```

Comment:
Concern was if some one is already using the nuget package , we probably don't want to mess with the defaults, Plus Vsix is exposing only few as defaults and we probably don't want to restrict nuget, VSIX will catch up to what is by default in nuget when it goes in box. 

Let us know if you think otherwise.